### PR TITLE
Iox #467 move iceoryx into iceoryx eclipse org

### LINKED ISSUE #467: Move iceoryx from eclipse to iceoryx-eclipse organization
@eclipse/technology-iceoryx-committers  
Currently, iceoryx is directly in the eclipse organization. For the future we want to create more repos shich belong to iceoryx so it make sense to create an own organization for aggregating all iceoryx repositories.

For that purpose was the `iceoryx-eclipse` organization created. Now we need to move the iceoryx repo into that organization.
The request for that is here: [https://bugs.eclipse.org/bugs/show_bug.cgi?id=565706](https://bugs.eclipse.org/bugs/show_bug.cgi?id=565706)

Updating the remote URL is done with that command once the transfer is done:
```
git remote set-url origin https://github.com/eclipse-iceoryx/iceoryx.git
```

### PR DESCRIPTION
**Pull request by dkroenke**
## Pre-Review Checklist for the PR Author

1. [x] Branch follows the naming format (`iox-#123-this-is-a-branch`)
1. [x] Commits messages are according to this [guideline][commit-guidelines]
    - [x] Commit messages have the issue ID (`iox-#123 commit text`)
    - [x] Commit messages are signed (`git commit -s`)
    - [x] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
1. [x] Update the PR title
   - Follow the same conventions as for commit messages
   - Link to the relevant issue
1. [x] Relevant issues are linked
1. [x] Add sensible notes for the reviewer
1. [x] All checks have passed
1. [x] Assign PR to reviewer

[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[eca]: http://www.eclipse.org/legal/ECA.php

## Notes for Reviewer
Updating all URLs in iceoryx to new organization

## Checklist for the PR Reviewer

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code according to our coding style and naming conventions
- [ ] Unit tests have been written for new behavior
- [ ] Public API changes are documented via doxygen
- [ ] Copyright owner are updated in the changed files
- [ ] PR title describes the changes

## Post-review Checklist for the PR Author

1. [ ] All open points are addressed and tracked via issues

## Post-review Checklist for the Eclipse Committer

1. [ ] All checkboxes in the PR checklist are checked or crossed out
1. [ ] Merge

## References

- Closes #467

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 <img src="https://user-images.githubusercontent.com/8661268/70233652-4aa6d180-175f-11ea-8524-2344e0d3935c.png" width="50%">
 </p>
 
-[![Build & Test](https://github.com/eclipse/iceoryx/workflows/Build%20&%20Test/badge.svg?branch=master)](https://github.com/eclipse/iceoryx/actions)
-[![Colcon build](https://github.com/eclipse/iceoryx/workflows/Colcon%20build/badge.svg?branch=master)](https://github.com/eclipse/iceoryx/actions)
-[![Gitter](https://badges.gitter.im/eclipse/iceoryx.svg)](https://gitter.im/eclipse/iceoryx)
+[![Build & Test](https://github.com/iceoryx-eclipse/iceoryx/workflows/Build%20&%20Test/badge.svg?branch=master)](https://github.com/eclipse-iceoryx/iceoryx/actions)
```
Comment (FerdinandSpitzschnueffler) [inline README.md:7]: Shouldn't it be ``eclipse-iceoryx``? Other way around the page is not found.

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
+[![Colcon build](https://github.com/iceoryx-eclipse/iceoryx/workflows/Colcon%20build/badge.svg?branch=master)](https://github.com/eclipse-iceoryx/iceoryx/actions)
```
Comment (FerdinandSpitzschnueffler) [inline README.md:8]: See comment above.

**Review by FerdinandSpitzschnueffler — APPROVED** [review]

**Review by elBoberido — APPROVED** [review]

**Comment by dkroenke** [comment]
@mossmaurice can you join yourself as reviewer?

**Comment by mossmaurice** [comment]
> @mossmaurice can you join yourself as reviewer?

Nope :cry:

**Review by elBoberido — APPROVED** [review]

File: GOVERNANCE.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ If you have points that you want to discuss, please send your points to the [mai
 
 ## Agenda
 
-1. General: Introduction of new participations, 42 mins
+1. General: Introduction of new participators, 42 mins
```
Comment (MatthiasKillat) [inline GOVERNANCE.md:67]: participants is more established as a word I guess, but participator exists as well and is synonymous (to my knowledge).

**Review by MatthiasKillat — APPROVED** [review]

**Pull request merged**